scripts/refresh_products_tags.js

```diff
@@ -0,0 +1,55 @@
+/* 
+  Name        : refresh_products_tags.js 
+  Description : Refresh products_tags collection by copying *_tags fields from products collection 
+  Version     : 1.0
+  Usage       : mongo refresh_products_tags.js
+  Add in crontab for daily refresh : 
+	00 02 * * * /usr/bin/mongo /etc/mongo/refresh_products_tags.js  >> /var/log/mongodb/refresh_products_tags.log
+*/
+
+db = db.getSiblingDB('off');

```

Comment:
is there a way to make the db name a parameter? we have separate names for OBF, OPF, OPFF